exercicios/para-casa/Biblioteca.py

```diff
@@ -0,0 +1,88 @@
+from Livro import Livro
+
+class Biblioteca:
+
+    def __init__(self):     # método construtor da classe
+        """Classe Biblioteca (tem como propriedade uma lista de livros)"""
+
+        # raise NotImplementedError     # levantando uma excessão para sinalizar que a classe não foi implementada
+
+        self.acervo = []
+        """acervo (lista): propriedade da classe Biblioteca"""
+    
+    def adicionar_livro(self, livro:Livro):     # parâmetro livro do tipo classe Livro
+        """Método para adiconar um livro ao acervo da Biblioteca"""
+
+        if not (isinstance(livro, Livro)):
+            # se a propriedade livro não for igual a classe Livro apresente mensagem de erro
+            raise TypeError (f"Esperado livro valor {livro} do tipo {type(livro)}")
+        
+        self.acervo.append(livro)
+            # adicona o livro ao acervo da Biblioteca
+
+    def exibir_acervo(self, livro:Livro):
+        """Método para exibir o acervo de livros da Biblioteca"""
+
+        for livro in self.acervo:
+            return livro
+            # para cada livro contido no acervo da Biblioteca: exiba os livros
+     
+    def remover_livro(self, livro:Livro):
+        """Método para remover um livro do acervo da Biblioteca"""
+
+        # para acessar os atributos da classe livro: livro_nomeatributo
+
+        if livro.tituloLivro in self.acervo and livro.estaEmprestado == False: 
```

Comment:
Muito bacana que fez a verificação se o livro já está emprestado antes de alterar o status novamente

Aqui só tem uma pequena correção: 
Vc está verificando se o seu parametro recebido (livro) está com o status de emprestado como falso.

é importante a gente validar com as propriedades da própria classe no caso seria: 

`and self.acervo[x].estaEmprestado == False: `

onde x é a posição do seu livro do acervo, e para encontrar o livro correto na lista tem algumas formas, a mais comum sendo o [uso da função index()](https://www.freecodecamp.org/news/python-find-in-list-how-to-find-the-index-of-an-item-or-element-in-a-list/)

E agora a dúvida: por que funcionou dessa forma? 
parabéns você acaba de conhecer mais uma mania do Python!! 
quando a gente cria um objeto e atribui esse objeto a outro eles guardam a mesma referencia, então quando altera um o outro também é alterado.
para se aprofundar nesse assunto aqui um artigo [Python - A diferença entre “Deep Copy e Shallow Copy”](https://gxara.medium.com/python-a-diferen%C3%A7a-entre-deep-copy-e-shallow-copy-2eab23e9b158#:~:text=Um%20shallow%20copy%2C%20ou%20%E2%80%9Cc%C3%B3pia,ser%C3%A1%20tamb%C3%A9m%20refletida%20no%20outro.)